styles.css

```diff
@@ -0,0 +1,1336 @@
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    height: 80px;
+    position: fixed;
+    z-index: 2;
+    top: 30px;
+    background-color: #fff;
+    border-bottom: 3px solid rgb(249, 246, 246);
+  }
+
+  .logo {
+    width: 50px;
+    height: 50px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 5px;
+    padding-left: 200px;
+    font-family: "Lato", sans-serif;
+  }
+
+  .logo img {
+    width: 50px;
+    height: 50px;
+  }
+
+  .menu-links {
+    padding-right: 200px;
+    font-family: "Lato", sans-serif;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .menu-links a:hover {
+    color: rgb(237, 76, 18);
+  }
+
+  .menu-links ul {
+    display: flex;
+    list-style: none;
+    flex-direction: row;
+    gap: 70px;
+    align-items: center;
+  }
+
+  .menu-link:nth-child(6) li {
+    border: 5px solid rgb(237, 76, 18);
+    padding: 10px;
+    color: rgb(237, 76, 18);
+  }
+
+  .landing {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: flex-start;
+    width: 100vw;
+    padding-top: 160px;
+    overflow-x: hidden;
+    background-repeat: no-repeat;
+    background-image: url(./images/bg.webp);
+    background-size: cover;
+    height: 100vh;
+    color: rgb(81, 80, 80);
+  }
+
+  .container {
+    width: 80%;
+    margin-left: 200px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .container h2 {
+    font-size: 2rem;
+    color: rgb(237, 76, 18);
+    font-family: 'Lato', sans-serif;
+  }
+
+  .container h1 {
+    font-size: 3rem;
+    color: rgb(237, 76, 18);
+    font-family: 'Lato', sans-serif;
+  }
+
+  .container p {
+    padding: 1.4rem 3.4rem;
+    font-size: 1rem;
+    display: block;
+    margin: 2rem;
+    border: 2px solid #fff;
+    width: 690px;
+    font-family: 'Lato', sans-serif;
+  }
+
+  .container h3 {
+    font-size: 2rem;
+    font-family: 'Lato', sans-serif;
+  }
+
+  .container h4 {
+    font-family: 'Lato', sans-serif;
+  }
+
+  .main {
+    background: url(./images/bgtexture.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 80vh;
+    overflow-x: hidden;
+    width: 100vw;
+  }
+
+  .main-program {
+    display: grid;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .main-program h2 {
+    font-size: 2rem;
+    color: #fff;
+    font-family: 'Lato', sans-serif;
+  }
+
+  .line {
+    width: 50px;
+    height: 1px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-left: 80px;
+    background-color: rgb(237, 76, 18);
+  }
+
+  .program-cards {
+    width: 80%;
+    margin-top: 100px;
+    margin-left: 200px;
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    gap: 1rem;
+  }
+
+  .card {
+    background-color: #19181a;
+    padding: 3rem 2rem;
+    text-align: center;

```

Comment:
The pages should look almost identical to the original design, outside the content differences. The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design. 
Kindly check some of the disparities shown belowand ensure they mtach the design :pray: 


![Screenshot from 2023-01-04 13-19-47](https://user-images.githubusercontent.com/65233860/210564474-71a90bf4-13dd-48ba-9333-2d3eaf67e0ed.png)


![Screenshot from 2023-01-04 13-19-56](https://user-images.githubusercontent.com/65233860/210564537-b6807bec-05eb-4744-b641-3b3f322d1c81.png)




![Screenshot from 2023-01-04 13-20-26](https://user-images.githubusercontent.com/65233860/210564572-6917a4f2-ced8-4bdf-99a7-772cdc9f2cdf.png)